congestion_control.py

```diff
@@ -0,0 +1,45 @@
+
+class CongestionControl:
```

Comment:
I think "Controller" rather than "Control" is a more standard name for this thing.